### ISSUE TITLE
The aiidalab-qe-workchain version is not updated when the QE app is updated via the app store

### ISSUE THREAD

**Issue by unkcpz** [aiidalab/aiidalab-qe#298]
**Describe the bug**
From AppStore, QeApp only updates the App itself, not the `aiidalab-qe-workchain`.

**To Reproduce**
1. Install an outdated version of QeApp first, say 22.05.0, then the `aiidalab-qe-workchain` version is also 22.05.0
2. Update the QeApp from App manager to `22.09.0`. 
3. Check the version by `pip freeze | grep aiidalab-qe-workchain` and it is still `22.05.0`

**Expected behavior**
The `aiidalab-qe-workchain` version should be always synchronized with QeApp version as designed.

**Comment by unkcpz** [comment]
I didn't test but if I understand correctly, appstore uses `aiidalab install` to install and update the app? Do it use `-U` option?

**Comment by csadorf** [comment]
The pip install as part of the app installation is carried out here: https://github.com/aiidalab/aiidalab/blob/4452bf3c537d73c0ee5409aafa21c761b4836c32/aiidalab/app.py#L264

It does not use the `-U/--upgrade` option and I assume that's indeed the issue. @unkcpz If I propose a change to the aiidalab package, would you be able to test whether that fixes the issue?

**Comment by unkcpz** [comment]
Thanks @csadorf, I think it is enough to know where the installation is implemented and how to test and fix that. 
Is that possible to pin the version in QeApp rather than using `-U` for this way of setting URL whl based pip dependencies?

**Comment by csadorf** [comment]
Wait, I'm just realizing that I did not assess the situation correctly. The work chain package is not automatically updated with a new version of the app, because it is pinned here: https://github.com/aiidalab/aiidalab-qe/blob/86607fbab00475cde3c8c65bca18cebf0298aa3b/setup.cfg#L26

To update the work chain package you need to follow the procedure outlined [here](https://github.com/aiidalab/aiidalab-qe/blob/master/src/README.md).

**Comment by csadorf** [comment]
@unkcpz Please note that the work chain dependency is automatically updated when performing a release with bumpver: https://github.com/aiidalab/aiidalab-qe/blob/86607fbab00475cde3c8c65bca18cebf0298aa3b/setup.cfg#L67

**Comment by unkcpz** [comment]
But that is the way for developers to update the workchain, no? For the users, the workchain should always have the version corresponding to the QeApp version, I assume.

**Comment by csadorf** [comment]
Yes and that's why this is kept in sync, see for example the latest release:
https://github.com/aiidalab/aiidalab-qe/blob/86607fbab00475cde3c8c65bca18cebf0298aa3b/setup.cfg#L26

**Comment by csadorf** [comment]
So, after offline debugging, we figured out that pip does not update the package if the version is the same. We need to update the package version as well with every release here: https://github.com/aiidalab/aiidalab-qe/blob/86607fbab00475cde3c8c65bca18cebf0298aa3b/src/setup.cfg#L3
